editor.planx.uk/src/components/Feedback/index.tsx

```diff
@@ -38,52 +39,33 @@ import {
 
 const Feedback: React.FC = () => {
   const [currentFeedbackView, setCurrentFeedbackView] =
-    useState<FeedbackView>("banner");
+    useState<FeedbackView | null>(null);
   const previousFeedbackView = usePrevious(currentFeedbackView);
-  const breadcrumbs = useStore((state) => state.breadcrumbs);
+  const [isDrawerOpen, setIsDrawerOpen] = useState(false);
 
-  useEffect(() => {
-    if (currentFeedbackView === "thanks") {
-      setCurrentFeedbackView("banner");
-    }
-  }, [breadcrumbs]);
-
-  const feedbackComponentRef = useRef<HTMLDivElement | null>(null);
-
-  const shouldScrollToView = () => {
-    switch (currentFeedbackView) {
-      case "banner":
-      case "thanks":
-        return false;
-      default:
-        return true;
-    }
-  };
-
-  useEffect(() => {
-    if (shouldScrollToView()) {
-      feedbackComponentRef.current?.scrollIntoView({
-        behavior: "smooth",
-        block: "start",
-      });
-    }
-  }, [currentFeedbackView]);
+  const closeDrawer = () => {
+    setIsDrawerOpen(false);
+    setCurrentFeedbackView(null);
+  }
 
   function handleFeedbackViewClick(event: ClickEvents) {
     switch (event) {
       case "close":
-        setCurrentFeedbackView("banner");
+        closeDrawer();
         break;
       case "back":
         setCurrentFeedbackView("triage");
         break;
       default:
+        setIsDrawerOpen(true);
         setCurrentFeedbackView(event);
         break;
     }
   }
 
   async function handleFeedbackFormSubmit(values: UserFeedback) {
+    if (!currentFeedbackView) throw Error("Cannot submit feedback in current view");
```

Comment:
This is just type narrowing, it's not possible to call this function whilst `currentFeedbackView` is `null`.